tls/extensions/s2n_server_supported_versions.c

```diff
@@ -38,13 +38,27 @@
  * Selected Version (2 byte)
  **/
 
-/* Leaving conn as input for future unification */
-int s2n_extensions_server_supported_versions_size(struct s2n_connection *conn)
+static int s2n_server_supported_versions_send(struct s2n_connection *conn, struct s2n_stuffer *out);
+static int s2n_server_supported_versions_recv(struct s2n_connection *conn, struct s2n_stuffer *in);
+
+const s2n_extension_type s2n_server_supported_versions_extension = {
+    .iana_value = TLS_EXTENSION_SUPPORTED_VERSIONS,
+    .is_response = true,
+    .send = s2n_server_supported_versions_send,
+    .recv = s2n_server_supported_versions_recv,
+    .should_send = s2n_extension_send_if_tls13_connection,
+    .if_missing = s2n_extension_noop_if_missing,
+};
+
+static int s2n_server_supported_versions_send(struct s2n_connection *conn, struct s2n_stuffer *out)
 {
-    return 6;
+    GUARD(s2n_stuffer_write_uint8(out, conn->server_protocol_version / 10));
+    GUARD(s2n_stuffer_write_uint8(out, conn->server_protocol_version % 10));
+
+    return S2N_SUCCESS;
```

Comment:
client side returns 0. Is there a reason for the difference?

Reply:
I just missed replacing the 0s on the client size. I'll fix that.